bootstrap.d/sys-devel.yml

```diff
@@ -61,6 +61,26 @@ tools:
     install:
       - args: ['make', 'install']
 
+  - name: host-mold
```

Comment:
I miss the label tag (if applicable) and the architecture flag. Can you add them?